tests/clients/test_ngsi_v2_cb.py

```diff
@@ -229,6 +231,147 @@ def test_entity_operations(self):
+                    }
+                ))
+                self.assertEqual(client.get_entity(entity_id=entity_update.id),
+                                 entity_update)
+                # 3) delete attribute
+                entity_update.delete_attributes(attrs=[attr_append])
+                client.update_entity(entity=entity_update)
+                self.assertNotEqual(client.get_entity(entity_id=entity_update.id),
+                                    entity_update)
+
+            # override_entity()
+            if "override_entity":
+                client.post_entity(entity=entity_init, update=True)
+                entity_override = entity_init.model_copy(deep=True)
+                # 1) append attribute
+                entity_override.add_attributes(attrs=[attr_append])
+                client.override_entity(entity=entity_override)
+                self.assertEqual(client.get_entity(entity_id=entity_override.id),
+                                 entity_override)
+                # 2) update existing attribute value
+                attr_append = NamedContextAttribute(**{
+                    "name": 'pressure',
+                    "type": 'Number',
+                    "value": 2050})
+                entity_override.update_attribute(attrs=[attr_append])
+                client.override_entity(entity=entity_override)
+                self.assertEqual(client.get_entity(entity_id=entity_override.id),
+                                 entity_override)
+                # 3) delete attribute
+                entity_override.delete_attributes(attrs=[attr_append])
+                client.override_entity(entity=entity_override)
+                self.assertEqual(client.get_entity(entity_id=entity_override.id),
+                                 entity_override)
+
+            # patch_entity
+            if "patch_entity":
+                client.post_entity(entity=entity_init, update=True)
+                entity_patch = entity_init.model_copy(deep=True)
+                # 1) append attribute
+                entity_patch.add_attributes(attrs=[attr_append])
+                client.patch_entity(entity=entity_patch)
+                self.assertEqual(client.get_entity(entity_id=entity_patch.id),
+                                 entity_patch)
+                # 2) update existing attribute value
+                attr_append = NamedContextAttribute(**{
+                    "name": 'pressure',
+                    "type": 'Number',
+                    "value": 2050})
+                entity_patch.update_attribute(attrs=[attr_append])
+                client.patch_entity(entity=entity_patch)
+                self.assertEqual(client.get_entity(entity_id=entity_patch.id),
+                                 entity_patch)
+                # 3) delete attribute
+                entity_patch.delete_attributes(attrs=[attr_append])
+                client.patch_entity(entity=entity_patch)
+                # TODO It is expected here not to pass the test
```

Comment:
@djs0109 The patch function can also delete attributes from an entity with the DELETE request

Reply:
Yes. Currently it is possible. But right now `patch_entity` works more like a `put_entity`. But I will remove the todos here and maybe open another issue